components/table/TaskTable.tsx

```diff
@@ -0,0 +1,75 @@
+import { ProjectDTO, TaskDTO } from '@/lib/types/dto';
+import { Button } from '@chakra-ui/button';
+import { Box, Flex, Heading } from '@chakra-ui/layout';
+import { Modal, ModalCloseButton, ModalContent, ModalHeader, ModalOverlay, Tbody } from '@chakra-ui/react';
+import { Table, Td, Th, Thead, Tr } from '@chakra-ui/table';
+import React, { useState } from 'react';
+import { TaskForm } from '../form/TaskForm';
+
+interface TaskTableProps {
+    project: ProjectDTO;
+    tasks: TaskDTO[];
+}
+
+function TaskTable({ project, tasks }: TaskTableProps): JSX.Element {
+    const [displayNewTaskForm, setDisplayNewTaskForm] = useState(false);
```

Comment:
I think we're better off with just a simple boolean state instead of some fancy chakra stuff.